ibis/expr/timecontext.py

```
@@ -152,6 +152,11 @@ def adjust_context(op: Node, timecontext: TimeContext) -> TimeContext:
     --------
     Adjusted time context
     """
+    raise com.IbisError(f'Unsupported input type for adjust context for {op}')
```

does this hit tests?

---

I also forgot why do we need to make this change, can you explain and perhaps add to comment?

---

Yeah this is to avoid fail silently because the default behavior is just return the input param itself. I added comment, also a test case in `test_timecontext.py`

---

Ok